src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,62 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        System.out.println("--------------Test create car----------------");
+        Manufacturer manufacturer = manufacturerService.get(9L);
+        Driver driverFirst = driverService.get(17L);
+        Driver driverSecond = driverService.get(18L);
+        Driver driverThird = driverService.get(15L);
+        Car car = new Car();
+        car.setManufacturer(manufacturer);
+        car.setDrivers(new ArrayList<>(Arrays.asList(driverFirst, driverSecond, driverThird)));
```

Comment:
could use List.of(elems...) method to make code easier

Reply:
I can't to do this, because I can't to remove or add objects from this list, it is ImmutableCollections.
https://prnt.sc/1u81l60

Reply:
ok. understand. than leave your code